app/Tasks/Task.php

```diff
@@ -90,4 +91,23 @@ public function runTask(string $title, \Closure $task) : int
             ? $task()
             : $this->command->runTask($title, $task);
     }
+
+    protected function getOutputLines(Builder $command) : array
+    {
+        return explode(PHP_EOL, $this->getOutput($command));
+    }
+
+    protected function getOutput(Builder $command) : string
+    {
+        $exit = $this->runCommandWithoutOutput($command);
+
+        if ($exit) {
+            throw new RuntimeException(vsprintf('Failed executing "%s" (exit code %d)', [
```

Comment:
this is going with a very different approach from the project, not that i don't like, just wondering 🤔

Reply:
Yeah, I could not wrap my mind around having a direct `int` return flow, so came up with the exception, that works very well in this instance. I like it too.